src/main/sensors/gyroanalyse.c

```diff
@@ -148,7 +148,7 @@ void gyroDataAnalyse(const gyroDev_t *gyroDev, biquadFilter_t *notchFilterDyn)
             sample = biquadFilterApply(&fftGyroFilter[axis], sample);
             gyroData[axis][fftIdx] = sample;
             if (axis == 0)
-                DEBUG_SET(DEBUG_FFT, 2, lrintf(sample * gyroDev->scale));
```

Comment:
and scale passed value

Reply:
Scale seems to be needed for raw data only, any after-filter data is already scaled in BF.

Reply:
It was related to comment above - scale data before passing into gyroDataAnalyseAdd, if necessary

Reply:
Seems like it's not necessary, thanks :-)
I've just pushed the amendments, have a look Petr